olp-cpp-sdk-dataservice-read/src/repositories/CatalogRepository.h

```diff
@@ -38,20 +38,20 @@ namespace repository {
 
 class CatalogRepository final {
  public:
-  static CatalogResponse GetCatalog(client::HRN catalog,
-                                    client::CancellationContext context,
-                                    CatalogRequest request,
-                                    client::OlpClientSettings settings);
-
-  static CatalogVersionResponse GetLatestVersion(
-      client::HRN catalog, client::CancellationContext cancellation_context,
-      CatalogVersionRequest request, client::OlpClientSettings settings);
-
-  static VersionsResponse GetVersionsList(
-      const client::HRN& catalog,
-      client::CancellationContext cancellation_context,
-      const VersionsRequest& request,
-      const client::OlpClientSettings& settings);
+  CatalogRepository(client::HRN catalog, client::OlpClientSettings settings);
+
+  CatalogResponse GetCatalog(CatalogRequest request,
+                             client::CancellationContext context);
+
+  CatalogVersionResponse GetLatestVersion(CatalogVersionRequest request,
```

Comment:
request is value, but in GetVersionsList we use reference. Can we do similar approach  in both cases?